### DISCUSSION TITLE
Geometry Import

### DISCUSSION THREAD

**Discussion by ShowerFeng** [firemodels/fds#12585] [Q&A] [answered]
Dear contributors,

Can FDS software import external geometries, such as from UG_NX or SolidWorks or common geometry files (SAT, STEP, STL), etc. ? Because unstructured geometry is in beta, it is easier to use specialized geometric modeling software for complex geometry.

Kindly regards,
FENG

**Comment by emanuelegissi** [comment, marked as answer]
FDS cannot directly import external geometries.

You may use preprocessing tools, as [Pyrosim](https://www.thunderheadeng.com/pyrosim) (MS Windows only, paid, professional support) or [BlenderFDS](https://www.blenderfds.org) (multi platform, open source and free, community support).

**Comment by emanuelegissi** [comment]
For full reference on third party tools see:
https://github.com/firemodels/fds/wiki/Third-Party-Tools
